core/src/main/resources/lib/form/entry.jelly

```diff
@@ -97,7 +97,7 @@ THE SOFTWARE.
       <j:otherwise>
         <j:choose>
           <j:when test="${attrs.help!=null}">
-            <div class="setting-main help-sibling" style="display: inline-flex; align-items: center;">
```

Comment:
checking this in an isolated way this does appear to fix it, but need to check if it impacts other components / why this was done

Reply:
I was looking around on other occasions and this seems to have no impact.

Reply:
it moves the help icon to the next line.